Pack20/Task01/Program.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Дан код программы на С++.  Вывести  все операции и частоту их использования в программе
+
+const string path = @"/home/smol/homework/Pack12/Task05/Program.cs";
+var text = File.ReadAllText(path);
+var words = text.Split(new char[]{' ', ',', ';', '[', ']', '{', '}', '(', ')', '/', '$', '-', '+', '<', '=', '>'}, StringSplitOptions.RemoveEmptyEntries );
+var operations = words.Where(item => char.IsUpper(item[0]) && char.IsAscii(item[0]));
+var uniqueOperations = operations.Distinct();
+foreach (var item in uniqueOperations)
+{
+    var counter = operations.Count(x => x == item);
```

Comment:
This Count actually enumerates operations. You perform this search in the foreach loop.
This is very bad approach, please read a bit about [computational complexity](https://en.wikipedia.org/wiki/Computational_complexity)

The best option is to create a dictionary { operator: count } (count equals 0 by default). Then you should enumerate all characters in the file and check if it is exists in the dictionary. It yes, then just increase it's value.